src/main/java/uk/gov/hmcts/reform/sscs/messaging/HmcHearingsEventTopicListener.java

```diff
@@ -0,0 +1,94 @@
+package uk.gov.hmcts.reform.sscs.messaging;
+
+import com.azure.messaging.servicebus.ServiceBusClientBuilder;
+import com.azure.messaging.servicebus.ServiceBusErrorContext;
+import com.azure.messaging.servicebus.ServiceBusException;
+import com.azure.messaging.servicebus.ServiceBusFailureReason;
+import com.azure.messaging.servicebus.ServiceBusProcessorClient;
+import com.azure.messaging.servicebus.ServiceBusReceivedMessage;
+import com.azure.messaging.servicebus.ServiceBusReceivedMessageContext;
+import lombok.Data;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+@Slf4j
+@Data
+@SuppressWarnings({
+    "PMD.LawOfDemeter",
+    "PMD.CompareObjectsWithEquals",
+    "PMD.CloseResource"
+})
+public final class HmcHearingsEventTopicListener {
```

Comment:
Can you please not make this final? We're probably going to need to mock this class for tests.